### PR TITLE
feat(tangle-dapp): Add sorting functionalities to tables on Nomination Page

### LINKED ISSUE #2369: [TASK] Implement Sorting Functionality for Tables on Nominations page
## Description
Enhance ux by implementing sorting capabilities for the all Validators table (Active/Waiting) within the Nomination page. This will improve usability and allow users to navigate and organize the validator data more effectively. 

### Sorting Features:
- Enable sorting on the following columns:
  - Identity, Effective Amount Staked , Self-Staked, Nominations, Commission
- Clickable column headers to toggle between ascending and descending order for sorting.
- Default sorting should be set on the "Effective Amount Staked" column in descending order to immediately highlight validators with the most significant stake.
  - Down Arrow (↓): This indicates descending order. When clicked, the table will sort the column from the largest or greatest value at the top to the smallest or least value at the bottom. 
  - Up Arrow (↑): This indicates ascending order. When clicked, the table will sort the column from the smallest or least value at the top to the largest or greatest value at the bottom.

### Example Screenshots
default is sorted by `effective amount staked`, selecting a table header item (i.e., `identity` in this case) will enable sorting
Image
table is now sorted by `identity` in alphabetical order
Image
selecting again will reverse the sorting order
Image
* above are examples of the interaction, same logic should be applied to all table header items. 
* similar logic should also be applied to `Nominations` and `Payouts` tables

### PR DESCRIPTION
**Pull request by vutuanlinh2k2**
## Summary of changes

_Provide a detailed description of proposed changes._

- Add sorting functionalities to the Nominations, Payouts, and Validators table
- Update the column align of those tables for better UI

### Proposed area of change

_Put an `x` in the boxes that apply._

- [ ] `apps/bridge-dapp`
- [ ] `apps/hubble-stats`
- [ ] `apps/stats-dapp`
- [x] `apps/tangle-dapp`
- [ ] `apps/testnet-leaderboard`
- [ ] `apps/faucet`
- [ ] `apps/zk-explorer`
- [ ] `libs/webb-ui-components`

### Reference issue to close (if applicable)

_Specify any issues that can be closed from these changes (e.g. `Closes #233`)._

- Closes #2369 

### Screen Recording

_If possible provide a screen recording of proposed change._

https://github.com/webb-tools/webb-dapp/assets/69841784/e335b73f-f0a7-4176-b352-66b4bec6a3d2

---

### Code Checklist

_Please be sure to add .stories documentation if any additions are made to `libs/webb-ui-components`._

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by drewstone** [comment]
Check link checker

**Review by AtelyPham — APPROVED** [review]
The CI is failing for @vutuanlinh2k2, otherwise looking good!👍

**Review by monaiuu — APPROVED** [review]
LGTM!

Btw, here's a related task for sorting functionality, which would be good to fix: https://github.com/webb-tools/webb-dapp/issues/2378

**Review by yurixander — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -67,7 +67,7 @@
   "dependencies": {
     "@actions/core": "^1.10.1",
     "@apollo/client": "^3.10.4",
-    "@buildwithsygma/sygma-sdk-core": "^2.10.0",
+    "@buildwithsygma/sygma-sdk-core": "^2.11.0",
```
Comment (saadjhk) [inline package.json:70]: `v2.11.0` is faulty, reverting to `2.10.0` should be fine

**Review by saadjhk — CHANGES_REQUESTED** [review]

**Pull request merged**